Library/Formula/shards.rb

```diff
@@ -0,0 +1,19 @@
+class Shards < Formula
+  desc "Dependency manager for the Crystal language"
+  homepage "https://github.com/ysbaddaden/shards"
+  url "https://github.com/ysbaddaden/shards/archive/v0.4.0.tar.gz"
+  sha256 "fff0f3c6562023fd279c80becf3683d2ac922ff097dda12f544f51f4b383ae5c"
+
+  depends_on "manastech/crystal/crystal-lang"
```

Comment:
You can't have mandatory cross-tap dependencies inside core Homebrew formula, and we won't accept optional cross-tap dependencies for a tap that Homebrew doesn't have supervision over. If it needs this dependency, you'd be better hosting this in a tap.